modules/ROOT/pages/anypoint-platform-cli-commands.adoc

```diff
@@ -1786,17 +1911,21 @@ Besides the `--name` option, this command also takes the default `--help`, `-f`/
 > runtime-mgr server token [options]
 ----
 
-This command gets server registration token. This token needs to be used to register a new server. +
-This command does not take any options, except for the default ones: `--help`, `-f`/`--fields` and `-o`/`--output`.
+This command gets server registration token. This token needs to be used to register a new server. 
```

Comment:
"gets a registration token" or "the registration token"

Reply:
Yep!